config/seata-config-core/src/main/java/org/apache/seata/config/ConfigurationChangeListener.java

```diff
@@ -40,9 +40,10 @@ public interface ConfigurationChangeListener {
     /**
      * The constant EXECUTOR_SERVICE.
      */
-    ExecutorService EXECUTOR_SERVICE = new ThreadPoolExecutor(CORE_LISTENER_THREAD, MAX_LISTENER_THREAD,
-        Integer.MAX_VALUE, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>(),
-        new NamedThreadFactory("configListenerOperate", MAX_LISTENER_THREAD));
+    ExecutorService EXECUTOR_SERVICE =
+        new ThreadPoolExecutor(CORE_LISTENER_THREAD, MAX_LISTENER_THREAD, MAX_LISTENER_THREAD, TimeUnit.MILLISECONDS,
```

Comment:
The event notification model is not suited for this threading model, the backpressure mode is better suited for dealing with direct interactions with users. Inputs external to the event model should not introduce unpredictable behavior to the core's operation.

Reply:
done